src/containers/Body/selectors.ts

```diff
@@ -4,8 +4,20 @@ import { createSelector } from 'reselect';
 import * as Models from 'models';
 import { State } from './models';
 
-const moviesResponse = (state): Models.MoviesResponseMap => (state as State).get('moviesResponse');
+const moviesResponse = (state): Models.MoviesResponseMap => {
+  return (state.get('bodyReducer') as State).get('moviesResponse');
```

Comment:
I think the word `Reducer` is redundant everywhere in names of store properties.